src/bundle/RepositoryInstaller/Command/InstallPlatformCommand.php

```diff
@@ -272,4 +283,38 @@ private function executeCommand(OutputInterface $output, $cmd, $timeout = 300)
             throw new \RuntimeException(sprintf('An error occurred when executing the "%s" command.', escapeshellarg($cmd)));
         }
     }
+
+    private function changeDefaultAdminPassword(InputInterface $input): int
+    {
+        $io = new SymfonyStyle($input, $this->output);
+        $password = $io->askHidden('Password (your input will be hidden)');
+
+        $commandInput = new ArrayInput([
+            'command' => 'ibexa:user:update-user',
+            'user' => 'admin',
+            '--password' => $password,
+        ]);
+
+        $commandInput->setInteractive(
+            $commandInput->isInteractive()
```

Comment:
You probably want to copy the interactivity of the "parent" command, not the same one :)
```suggestion
            $input->isInteractive()
```

Reply:
My mistake: https://github.com/ibexa/core/pull/525/commits/a0829efa4be079df48ea58fe6a866dfb8b03731d.